fsspec/asyn.py

```diff
@@ -535,21 +565,38 @@ async def _get(
         constructor, or for all instances by setting the "gather_batch_size" key
         in ``fsspec.config.conf``, falling back to 1/8th of the system limit .
         """
-        from fsspec.implementations.local import LocalFileSystem, make_path_posix
+        from .implementations.local import (
```

Comment:
Do we get circular imports if we move all these from the methods to the top of the module?

Reply:
Moving the imports to the top works fine locally, I've pushed a commit to check in CI.